sources/filesystem/__init__.py

```diff
@@ -0,0 +1,138 @@
+"""Those resources collect filepaths from local folder or Google Drive folder to destinations"""
+import logging
+from pathlib import Path
+from typing import Optional, Sequence, Dict, Any
+
+import dlt
+from dlt.extract.source import TDataItem
+
+from .helpers import build_service, download_file_from_google_drive, get_files_uris
+from .settings import (
+    AUTHORIZED_USER_PATH,
+    DATA_DIR,
+    FOLDER_IDS,
+    STORAGE_FOLDER_PATH,
+    ClIENT_SECRET_PATH,
+)
+
+
+@dlt.resource(write_disposition="replace")
```

Comment:
I think it should be `selected=False` by default. then the data will be passed to transformer and not stored as separate table

Reply:
it was stored because I returned it in unstructured_source. I was planned